configs/mini_pupper_config/package.xml

```diff
@@ -11,10 +11,9 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>launch_ros</build_depend>
-  <!-- <exec_depend>roslaunch</exec_depend>
-  <exec_depend>rviz</exec_depend>
   <exec_depend>champ_base</exec_depend>
-   -->
+  <exec_depend>launch_ros</exec_depend>
+  <exec_depend>rviz</exec_depend>
```

Comment:
i think we need put it rviz2. I didn't check early. Just found one reference: https://github.com/ROBOTIS-GIT/turtlebot3/blob/ros2/turtlebot3_bringup/package.xml

Reply:
Thanks for pointing it out. Fixed in https://github.com/CullenSUN/chvmp_robots/pull/1/commits/5fabcf3de37a09b2b9b89f8672e30c98b4856be9